test/e2e/nodes_test.go

```diff
@@ -27,9 +29,22 @@ var _ = Describe("Nodes", func() {
 			for _, node := range nodes {
 				key := types.NamespacedName{Namespace: namespace, Name: node}
 				nodeNetworkState := nodeNetworkState(key)
-				//TODO: exec nmstatectl is not in place let's just compare
-				//      with the stuff we have harcoded there
-				Expect(nodeNetworkState.Status.CurrentState).To(MatchYAML("interfaces: []"))
+				currentStateYaml := nodeNetworkState.Status.CurrentState
+				Expect(currentStateYaml).ToNot(BeEmpty())
```

Comment:
please make the code breathe. empty line after every

```
x := X()
Expect(...)
```

Reply:
Done